Assets/Gothic-UnZENity-Core/Scripts/Manager/NpcHelper.cs

```diff
@@ -32,6 +32,7 @@ public static void CacheHero(GameObject playerGo)
             // Set data for NPC.
             var npcInstance = (NpcInstance)GameData.GothicVm.GlobalHero;
             playerProperties.NpcInstance = npcInstance;
+            playerProperties.Head = Camera.main.transform;
```

Comment:
Thanks for adding this. I checked it and this transform seems to be exactly at the height of the player. Finally. :-)